script/clone_job.pl

```diff
@@ -41,7 +41,7 @@ =head1 OPTIONS
 
 =item B<--dir> DIR
 
-specify directory where the iso is stored (default /var/lib/openqa/factory/iso/)
+specify directory where the iso is stored (default /var/lib/openqa/factory)
```

Comment:
Please change it to `specify directory where test assets are stored (default /var/lib/openqa/factory`